__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const db = require('../db/connection.js')
+const seed = require('../db/seeds/seed.js')
+const request = require('supertest')
+const app = require('../app')
+const testData = require('../db/data/test-data');
+
+afterAll(() => {
+    return db.end()
+})
+beforeEach(() => {
+    return seed(testData)
+})
+
+describe(' GET /api/topics', () => {
+    test('returns a status 200', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body : topics}) => {
+            topics.forEach((topic) => {
```

Comment:
Good test but is missing a check for the length of the array, need to check it's length in case it's zero.